### PR TITLE
Add AV audio skin story - AMP player

### LINKED ISSUE #2264: Add an AV audio skin story to Psammead
**Describe the bug**
This is currently no story for Media Player audio skin AMP. 

**Expected behaviour**
Add a story for the Media Player component that has the skin audio and uses the AMP player. 

**Screenshots**
image

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by ibMadbouly**
Resolves #2264 

**Code changes:**

- in `packages/components/psammead-media-player/src/index.stories.jsx` added new story entry for AV audio skin using `AmpMediaPlayer`

**Testing notes** 

1. `npm run storybook`.
2. Check on storybook we got the `Audio Skin AMP` 
image

 

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-media-player/src/index.stories.jsx [hunk 1]
```diff
@@ -27,3 +27,12 @@ storiesOf('Components|Media Player', module).add('Audio Skin', () => (
     skin="audio"
   />
 ));
+
+storiesOf('Components|Media Player', module).add('Audio Skin AMP', () => (
+  <AmpMediaPlayer
+    isAmp
```
Comment (pjlee11) [inline packages/components/psammead-media-player/src/index.stories.jsx]: The props for the AmpMediaPlayer are different to this. Also you need to add the `ampDecorator`, see line 13 to 21

**Review by pjlee11 — CHANGES_REQUESTED** [review]
Story needs to be changed

File: packages/components/psammead-media-player/src/index.stories.jsx (hunk 1 above)
Comment (ibMadbouly) [reply packages/components/psammead-media-player/src/index.stories.jsx]: @pjlee11  Done

File: packages/components/psammead-media-player/src/index.stories.jsx [hunk 2]
```diff
@@ -27,3 +27,14 @@ storiesOf('Components|Media Player', module).add('Audio Skin', () => (
...
   />
 ));
+
+storiesOf('Components|Media Player', module)
+  .add(ampDecorator)
+  .add('Audio Skin AMP', () => (
+    <AmpMediaPlayer
+      isAmp
+      src="https://www.test.bbc.co.uk/ws/av-embeds/articles/c3wmq4d1y3wo/p01k6msp"
+      placeholderSrc="https://ichef.bbci.co.uk/news/640/cpsdevpb/4eb7/test/ba7482d0-cca8-11e8-b0bf-f33155223fc4.jpg"
```
Comment (pjlee11) [inline packages/components/psammead-media-player/src/index.stories.jsx]:
```suggestion
      placeholderSrc="https://news.files.bbci.co.uk/include/articles/public/images/audio-player-placeholder.png"
```

The audio placeholder image is now available 😄

File: packages/components/psammead-media-player/src/index.stories.jsx [hunk 3]
```diff
@@ -27,3 +27,14 @@ storiesOf('Components|Media Player', module).add('Audio Skin', () => (
     skin="audio"
   />
 ));
+
+storiesOf('Components|Media Player', module)
+  .add(ampDecorator)
+  .add('Audio Skin AMP', () => (
+    <AmpMediaPlayer
+      isAmp
+      src="https://www.test.bbc.co.uk/ws/av-embeds/articles/c3wmq4d1y3wo/p01k6msp"
```
Comment (pjlee11) [inline packages/components/psammead-media-player/src/index.stories.jsx]:
```suggestion
      src="https://www.test.bbc.com/ws/av-embeds/media/bbc_korean_radio/liveradio/amp"
```

Can this be changed to the liveradio player?

Comment (ibMadbouly) [reply packages/components/psammead-media-player/src/index.stories.jsx]: Done 👍

File: packages/components/psammead-media-player/src/index.stories.jsx (hunk 2 above)
Comment (ibMadbouly) [reply packages/components/psammead-media-player/src/index.stories.jsx]: Done 👍

**Review by pjlee11 — APPROVED** [review]
LGTM 👍

**Review by ryanmccombe — APPROVED** [review]

**Review by dr3 — APPROVED** [review]

**Pull request merged**